apps/server/src/controllers/ontimeController.validate.ts

```diff
@@ -166,3 +171,137 @@ export const validateLoadProjectFile = [
     next();
   },
 ];
+
+/**
+ * @description Validates the filenames for duplicating a project.
+ */
+export const validateProjectDuplicate = [
+  body('projectFilename')
+    .exists()
+    .withMessage('Project filename is required')
+    .isString()
+    .withMessage('Project filename must be a string'),
```

Comment:
makes sense, do you have a length max in mind?

OS limits apparently are 255

```
BTRFS   255 bytes
exFAT   255 UTF-16 characters
ext2    255 bytes
ext3    255 bytes
ext3cow 255 bytes
ext4    255 bytes
FAT32   8.3 (255 UCS-2 code units with VFAT LFNs)
NTFS    255 characters
XFS     255 bytes
```

Reply:
oh, I meant the other way, we want to check that the name has at least 1 (?) character.
I wonder if an empty string would pass the isString validation.

I dont think we should worry about the max characters

Reply:
makes sense,

I added a `min: 1`, so it should. I'll double check either way.

Maybe keeping `max: 255` isn't a bad idea, just in case, since the OS might not be able to handle it